src/index.test.tsx

```diff
@@ -220,6 +220,26 @@ describe("useMasonry()", () => {
     }
   });
 
+  it("should render children sorted by index", () => {
+    const hook = renderBasicMasonry({
+      items: [
+        { id: 0, height: 100 },
+        { id: 1, height: 200 },
+        { id: 2, height: 300 },
+      ],
+      overscanBy: 1,
+      itemHeightEstimate: 100,
+    });
```

Comment:
I don't really like this setup step, as it's not explicit that the order of items is not sorted by index, I am going to replace it with a faked positioned with explicit order